### PR TITLE
dev-embedded/stm32flash: Update to release 0.6

### PR DESCRIPTION
**Pull request by EvanTheB**
Included patches to fix
https://sourceforge.net/p/stm32flash/tickets/98/.

Signed-off-by: Evan Benn <evanbenn@chromium.org>

### CODE REVIEW

**Comment by EvanTheB** [comment]
I tested this build on ChromeOS, only on amd64:

https://chromium-review.googlesource.com/c/chromiumos/overlays/portage-stable/+/3146698
https://chromium-review.googlesource.com/c/chromiumos/overlays/portage-stable/+/3153198

Not sure what else I need to do to land this here, please let me know.

File: dev-embedded/stm32flash/stm32flash-0.6.ebuild [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```
Comment (juippis) [inline dev-embedded/stm32flash/stm32flash-0.6.ebuild:4]: 8 should work already here.

File: dev-embedded/stm32flash/Manifest [hunk 2]
```diff
@@ -1 +1,2 @@
 DIST stm32flash-0.5.tar.gz 41087 BLAKE2B 581ef883537783fafcff8a7471a697f50929b572d847791d692799040fd54845a67a7b6b608401df24f05810ae81b7e19df646f2835bc3d7cd35888d8649be6a SHA512 a01875a315d92d5e347372b736f2afa2f633851bf13af1326341c0c91e17cef54b6730ed1279280e86d3797033c856dfc25e8c595e010772b9287aaf965273f9
+DIST stm32flash-0.6.tar.gz 376186 SHA512 2d92ecf4f82b8a702b596cbf3c4ff594f67083f7dcec82c064555f73b47834b41a918efa1846e47fc8740ae63fa818b16dfb5b38fa242c17f465666a651eee3a
```
Comment (juippis) [inline dev-embedded/stm32flash/Manifest:2]: We still use both SHA512 and BLAKE2B to verify hashes. You should be able to generate this via either
`repoman manifest`, 
`pkgdev commit`, or
`ebuild stm32flash-0.6.ebuild manifest` when positioned in ::gentoo repo dir.

**Review by juippis — COMMENTED** [review]
Please fix the CI issue (correct manifest hashes) and I'd prefer if you bumped and deleted previous version in separate commits. So this should be 2 commits instead of 1;
add 0.6
remove 0.5.

**Review by juippis — APPROVED** [review]

**Comment by juippis** [comment]
Disregard that ^ I can't push your commit even if I fix it afterwards, and with GLEP-76 rebasing my fix to your commit is "illegal". 

So please fix this contribution :)

```
remote: FATAL: VREF/proj-gentoo-05-manifest: helper program exit status 256
remote: Disallowed hash set in Manifest!
remote:  commit: a0db3e0407bf53d5a8fb50a97d68f25373556fe1
remote:    file: dev-embedded/stm32flash/Manifest
remote:   entry: DIST stm32flash-0.6.tar.gz 376186 SHA512 2d92ecf4f82b8a702b596cbf3c4ff594f67083f7dcec82c064555f73b47834b41a918efa1846e47fc8740ae63fa818b16dfb5b38fa242c17f465666a651eee3a
remote: error: hook declined to update refs/heads/master
To git.gentoo.org:repo/gentoo.git
 ! [remote rejected]         master -> master (hook declined)
error: failed to push some refs to 'git.gentoo.org:repo/gentoo.git'
```

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**